src/Lib/globals.cpp

```diff
@@ -0,0 +1,21 @@
+
+#include "globals.hpp"
+
+#ifndef PACKAGE_VERSION
+#include "config.h"
+#endif
+
+std::string GetVersion(void)
```

Comment:
Actually I think having a global GetVersion method is kinda ambiguous, there are many things that can be versioned, like network protocols etc. Maybe change the name or add a class and make this a static method like Package::GetVersion?

Reply:
> Actually I think having a global GetVersion method is kinda ambiguous, there are many things that can be versioned, like network protocols etc. Maybe change the name or add a class and make this a static method like Package::GetVersion?

Sounds reasonable. Done.